### ISSUE TITLE
tilelive.serialize() should throw on invalid Tile obj

### ISSUE THREAD

**Issue by ian29** [mapbox/tilelive.js#117]
if you feed `tilelive.serialize()` some poorly formatted Tiles, it sliently skips over them. instead of silently skipping bad stream input, tilellive should probably error or at least warn when you're feeding it bad stuff.

**Comment by ian29** [comment]
erps, meant to point at this bit in particular:
https://github.com/mapbox/tilelive.js/blob/master/lib/stream-serialize.js#L22

**Comment by rclark** [comment]
I agree -- I think the stream should emit an error in this situation, just like regular streams do if `objectMode: false` and it receives a non-string/buffer.
